src/deye_mqtt.py

```diff
@@ -118,16 +118,18 @@ def __build_topic_name(self, logger_topic_prefix: str, topic_suffix: str) -> str
     def __map_logger_index_to_topic_prefix(self, logger_index: int):
         return str(logger_index) if logger_index > 0 else ""
 
+    def build_topic_name(self, logger_index: int, topic_suffix: str) -> str:
+        logger_topic_prefix = self.__map_logger_index_to_topic_prefix(logger_index)
```

Comment:
What do you think to include `__map_logger_index_to_topic_prefix()` directly instead of calling as a function as it's only used to create the topic?

Reply:
It's used twice in this class actually.